tests/flashcore/test_session_database.py

```
@@ -0,0 +1,402 @@
+        """Test getting reviews for session with no reviews."""
+        created_session = in_memory_db.create_session(sample_session)
+        reviews = in_memory_db.get_reviews_for_session(created_session.session_uuid)
+
+        assert reviews == []
+
+    def test_get_reviews_for_session_database_error(self, in_memory_db):
+        """Test handling of database errors when getting reviews for session."""
+        # Close the database connection to simulate error
+        in_memory_db.close_connection()
+
+        with pytest.raises(Exception):  # Could be ReviewOperationError or CardOperationError
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)